files/sanity_check.py

```diff
@@ -129,13 +126,12 @@ def test_ports(ports):
 def test_commands(commands):
     response = []
     for command in commands:
-        if not check_command(command['command']):
-            response.append(SanityCheckResult(False, command['message']))
-        else:
-            message = format("Command {} returned exit status 0".format(
-                command['command']
-            ))
-            response.append(SanityCheckResult(True, message))
+        command_passed, command_output = check_command(command['command'])
+        command_message = (
+            "Command {} returned exit status 0".format(command['command'])
```

Comment:
@UmanShahzad if command failed most likely (or absolutely always?) it returns non-zero status code. This message will be quite misleading.

And in general, it would look like:

    * "Command bash returned exit code 0" if `command_passed` is true
    * "Command Exception: something very bad happened + giant stacktrace .... snip .... few dozen lines later returned exit status 0" if `command_passed` is false.

Maybe something like

```
if command_passed:
    command_message = "Command {} returned status code 0"
else:
    command_message=  "Command {} returned status code {}; stdout/stderr was {}".format(...)
```

Reply:
@e-kolpakov I'm confused what you mean by this. The outputs look like this in the report:

Passed:
```
#. OK Command rabbitmqctl node_health_check returned exit status 0
Timeout: 70.0 seconds
Checking health of node rabbit@rabbitmq
Health check passed
#. OK Enough free space /. Free space is 85.0%
#. OK Port localhost:5671 is getting connections
#. OK Port localhost:15671 is getting connections
```

Not passed:
```
#. ERROR RabbitMQ health check not passing
Timeout: 70.0 seconds
Checking health of node rabbit@rabbitmq
Error: {killed,{gen_server2,call,[<10994.27345.1912>,{info,[pid]},infinity]}}
#. OK Enough free space /. Free space is 85.0%
#. OK Port localhost:5671 is getting connections
#. OK Port localhost:15671 is getting connections
```

Reply:
@UmanShahzad right, now I see it. This code is a bit confusing though - maybe something more traditional would be less confusing? Like

```
if success:
    message = "Command {} returned 0"
else:
    message = command['message'] 
```

Reply:
This is actually idiomatic in Python but since it did throw someone off, it might happen again so OK.

Reply:
:+1: makes sense.